meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -2,3 +2,26 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 EXTRA_OENPM:witherspoon-tacoma = "-- --mode ibm"
 EXTRA_OENPM:p10bmc = "-- --mode ibm"
+
+inherit obmc-phosphor-systemd
+
+SRC_URI:append:p10bmc = " \
+        file://changeLogo.sh \
+        file://change-logo.service \
+        file://inspur-login-logo.svg \
+        file://inspur-logo-header.svg \
+        file://blankLogo.svg \
+        "
+
+SYSTEMD_SERVICE:${PN}:append:p10bmc = " change-logo.service"
+
+do_install:append:p10bmc() {
+    gzip ${WORKDIR}/inspur-login-logo.svg
+    gzip ${WORKDIR}/inspur-logo-header.svg
```

Comment:
I've gotten a mix of re-compile issues with this code:

| gzip: /home/andrewg/Code/openbmc-ghe/build/p10bmc/tmp/work/all-openbmc-linux/webui-vue/1.0+gitAUTOINC+03be4abc07-r1/inspur-login-logo.svg.gz already exists;  not overwritten

and

| gzip: /home/andrewg/Code/openbmc-ghe/build/p10bmc/tmp/work/all-openbmc-linux/webui-vue/1.0+gitAUTOINC+03be4abc07-r1/inspur-login-logo.svg: No such file or directory

I think the second one is why the check was in to see if the files existed.